regression/goto-analyzer/value-set-function-pointers-simple/main.c

```diff
@@ -1,7 +1,6 @@
 #include <assert.h>
 
 typedef int (*fptr_t)(int);
-fptr_t fun_global, fun_global_show;
```

Comment:
Would it be possible to add the reduced version of this as another test (can be another `.c` and another `.desc` in the same directory ) rather than just reducing this one?